.github/actions/push-to-ecr/action.yml

```diff
@@ -0,0 +1,130 @@
+name: 'Push Image to ECR'
+description: 'Authenticate with ECR, validate tags, and push a Docker image'
+
+inputs:
+  ecr-repository:
+    description: 'ECR repository name (e.g., s/heroku/widgetsrus/widgetsrus)'
+    required: true
+  image-tag:
+    description: 'Image tag to use (e.g., commit SHA or version number)'
+    required: true
+  image-name:
+    description: 'Local Docker image name to push (defaults to ecr-repository:image-tag)'
```

Comment:
is this true? it doesn't seem to be implemented this way and `image-name` isn't passed in, so will default to the `ARCH`.